controllers/chat/moveToSign.js

```diff
@@ -117,10 +97,15 @@ const moveToSign = async (req, res) => {
       limit: 100
     });
 
+    const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
+      await BetterSocialCore.chat.updateBetterChannelMembers(newChannel, createdChannel, true, {
+        channel_type: isContainAnonimous ? CHANNEL_TYPE.ANONYMOUS : CHANNEL_TYPE.CHAT
+      });
+
     const response = {
-      ...createdChannel,
-      better_channel_members_object: newStateMemberWithAnonInfo,
-      better_channel_members: Object.values(newStateMemberWithAnonInfo),
+      ...updatedChannel,
+      better_channel_members_object: betterChannelMemberObject,
+      better_channel_members: betterChannelMember,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The response object structure has been changed significantly. Ensure that these changes do not break any client-side code that depends on the previous structure of the response object.
```diff
-     const response = {
-      ...createdChannel,
-      better_channel_members_object: newStateMemberWithAnonInfo,
-      better_channel_members: Object.values(newStateMemberWithAnonInfo),
-       messageHistories: messageHistory.results
-     };
+     const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
+       await BetterSocialCore.chat.updateBetterChannelMembers(newChannel, createdChannel, true, {
+         channel_type: isContainAnonimous ? CHANNEL_TYPE.ANONYMOUS : CHANNEL_TYPE.CHAT
+       });
+ 
+      const response = {
+       ...updatedChannel,
+       better_channel_members_object: betterChannelMemberObject,
+       better_channel_members: betterChannelMember,
+       messageHistories: messageHistory.results
+     };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->